tactics/elim.ml

```diff
@@ -217,52 +214,3 @@ let h_decompose l c = decompose_these c l
 let h_decompose_or = decompose_or
 
 let h_decompose_and = decompose_and
-
-(* The tactic Double performs a double induction *)
-
-let induction_trailer abs_i abs_j bargs =
-  tclTHEN
-    (tclDO (abs_j - abs_i) intro)
-    (onLastHypId
-       (fun id ->
-          Proofview.Goal.enter begin fun gl ->
-          let idty = pf_get_type_of gl (mkVar id) in
-          let fvty = global_vars (pf_env gl) (project gl) idty in
-          let possible_bring_hyps =
-            (List.tl (nLastDecls gl (abs_j - abs_i))) @ bargs
-          in
-          let (hyps,_) =
-            List.fold_left
-              (fun (bring_ids,leave_ids) d ->
-                 let cid = NamedDecl.get_id d in
-                 if not (List.mem cid leave_ids)
-                 then (d::bring_ids,leave_ids)
-                 else (bring_ids,cid::leave_ids))
-              ([],fvty) possible_bring_hyps
-          in
-          let ids = List.rev (ids_of_named_context hyps) in
-          (tclTHENLIST
-            [revert ids; elimination_then (fun _ -> tclIDTAC) id])
-          end
-          ))
-
-let double_ind h1 h2 =
-  Proofview.Goal.enter begin fun gl ->
-  let abs_i = depth_of_quantified_hypothesis true h1 gl in
```

Comment:
You can also probably remove `depth_of_quantified_hypothesis` from the `Tactics` interface. It's not used anywhere else in Coq and unlikely to be used in a plugin given its specificity.

Reply:
Got it.  Update.